gtsfm/loader/hilti_loader.py

```diff
@@ -301,7 +308,7 @@ def image_from_rig_and_camera(self, rig_index: int, camera_idx: int) -> int:
     def get_relative_pose_priors(self, pairs: List[Tuple[int, int]]) -> Dict[Tuple[int, int], PosePrior]:
         pairs = set(pairs)
         # For every rig index, add a "star" from camera 2 to 0,1,3,4:
-        for rig_index in range(self.num_rig_poses):
+        for rig_index in range(0, self.num_rig_poses, SUBSAMPLE_FACTOR if self._subsample else 1):
```

Comment:
why are we subsampling the relative between factors?

Reply:
AFAIK we only subsample the indices at which we match, but keep these relative poses? So that final result is at 10Hz?

Reply:
No these are the start factors. If I add them, the cameras would just be dangling cameras without any contribution to the factor graph.

All Cam2-cam2 edges are coming from retriever and are at 10Hz.